docs/tutorial/file.rst

```diff
@@ -148,6 +151,40 @@ Once you access the raw data, those are regular arrays which use regular array i
     channel_slice = file.force1x['1.5s':'20s']  # timestamps
     data_slice = file.force1x.data[20:40]  # indices into the array
 
+Downsampling
+^^^^^^^^^^^^
+
+A slice can be downsampled using various methods.
+
+To downsample to a specific frequency use `downsampled_to` with the desired frequency in Hz::
+
+    channel = file.force1x # original frequency 78125 Hz
+    timestep = np.diff(channel.timestamps[:2]) * 1e-9        # timestep 12.8 us
+
+    ds_channel = channel.downsampled_to(3125)
+    ds_timestep = np.diff(ds_channel.timestamps[:2]) * 1e-9  # timestep 320 us
+
+This method can only be applied successfully if the ratio of the current and target frequencies
+is an integer. To automatically find the nearest higher frequency that will fulfill this requirement,
```

Comment:
Might be nice to specify what successfully means here:

`By default, this method will take the mean of every N samples where N is defined as the ratio between the two sampling times. This can cause issues when N isn't an integer, leading to an unequal number of points contributing to each point in the downsampled channel.` and then continue with the To automatically you already have